<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Anulekha-HP" name="Default test" tests="1" failures="1" timestamp="12 Feb 2017 06:53:09 GMT" time="19.426" errors="0">
  <testcase name="@AfterMethod tearDown" time="0.0" classname="testCases.VerifyLoginPageWithReports">
    <failure type="java.lang.RuntimeException" message="Process refused to die after 10 seconds, and couldn&amp;apos;t taskkill it: Unable to find executable for: taskkill">
      <![CDATA[java.lang.RuntimeException: Process refused to die after 10 seconds, and couldn't taskkill it: Unable to find executable for: taskkill
	at org.openqa.selenium.os.ProcessUtils.killWinProcess(ProcessUtils.java:143)
	at org.openqa.selenium.os.ProcessUtils.killProcess(ProcessUtils.java:81)
	at org.openqa.selenium.os.UnixProcess$SeleniumWatchDog.destroyHarder(UnixProcess.java:247)
	at org.openqa.selenium.os.UnixProcess$SeleniumWatchDog.access$200(UnixProcess.java:201)
	at org.openqa.selenium.os.UnixProcess.destroy(UnixProcess.java:125)
	at org.openqa.selenium.os.CommandLine.destroy(CommandLine.java:155)
	at org.openqa.selenium.firefox.FirefoxBinary.quit(FirefoxBinary.java:246)
	at org.openqa.selenium.firefox.internal.NewProfileExtensionConnection.quit(NewProfileExtensionConnection.java:194)
	at org.openqa.selenium.firefox.FirefoxDriver$LazyCommandExecutor.quit(FirefoxDriver.java:364)
	at org.openqa.selenium.firefox.FirefoxDriver.stopClient(FirefoxDriver.java:310)
	at org.openqa.selenium.remote.RemoteWebDriver.quit(RemoteWebDriver.java:529)
	at Factory.BrowserFactory.closeBrowser(BrowserFactory.java:40)
	at testCases.VerifyLoginPageWithReports.tearDown(VerifyLoginPageWithReports.java:70)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:564)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:213)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:786)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: java.lang.NullPointerException: Unable to find executable for: taskkill
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:251)
	at org.openqa.selenium.os.UnixProcess.<init>(UnixProcess.java:62)
	at org.openqa.selenium.os.CommandLine.<init>(CommandLine.java:38)
	at org.openqa.selenium.os.WindowsUtils.killPID(WindowsUtils.java:178)
	at org.openqa.selenium.os.ProcessUtils.killWinProcess(ProcessUtils.java:138)
	... 38 more
]]>
    </failure>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase name="testLoginPage" time="2.634" classname="testCases.VerifyLoginPageWithReports">
    <failure type="java.lang.NoClassDefFoundError" message="org/apache/xmlbeans/XmlException">
      <![CDATA[java.lang.NoClassDefFoundError: org/apache/xmlbeans/XmlException
	at dataProvider.ExcelDataProvider.<init>(ExcelDataProvider.java:18)
	at Factory.DataProviderFactory.getExcel(DataProviderFactory.java:15)
	at testCases.VerifyLoginPageWithReports.testLoginPage(VerifyLoginPageWithReports.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
Caused by: java.lang.ClassNotFoundException: org.apache.xmlbeans.XmlException
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:331)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 27 more
]]>
    </failure>
  </testcase> <!-- testLoginPage -->
</testsuite> <!-- Default test -->
